tolgee-action/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Tolgee Actions'
+description: 'Extract metadata and data from Tolgee project'
+inputs:
+  tolgee_secret:
+    description: 'Valid API key for the Tolgee project to use'
+    required: true
+  author_name:
+    description: 'Name used when updating locale data'
+    required: false
+    default: 'github-actions[bot]'
+  author_email:
+    description: 'Email used when updating locale data'
+    required: false
+    default: 'github-actions[bot]@users.noreply.github.com'
+  commit_message:
+    description: 'Commit message used when updating locale data'
+    required: false
+    default: ''
+runs:
```

Comment:
Maybe we can use bun for this instead of node? That we we can skip all the transpile stuff and just natively run the typescript script.

Reply:
I'm not aware of that possible, node20 is what is used in the gha environment.

Reply:
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing-for-javascript-actions

Reply:
Yes they try to 'help' you. But we can probably just do a custom action and run using bun
https://bun.sh/guides/runtime/cicd

We can definetely do it using a composite action as well.